site/docs/v1/tech/themes/_theme-troubleshooting.adoc

```diff
@@ -20,4 +20,4 @@ Anytime a request is made to a themed page and we are unable to identify the ten
 * Any themed pages such as /password/forgot when a `client_id` is not provided.
```

Comment:
While you are on these pages should this be `` `/password/forgot` `` wrapped in backticks?